src/components/select/_macro.njk

```diff
@@ -21,7 +21,7 @@
         <select
             id="{{ params.id }}"
             name="{{ params.name }}"
-            class="input input--select {{ params.classes }}"
+            class="input input--select{% if params.classes %} {{ params.classes }}{% endif %}{% if params.select.error %} input--error{% endif %}"
```

Comment:
Does it need the `select`?
```suggestion
            class="input input--select{% if params.classes %} {{ params.classes }}{% endif %}{% if params.error %} input--error{% endif %}"
```

Reply:
Yeah I needed a different param because setting `params.error` would add all the panel styling etc and we would end up with the double error panel situation again. It was the best solution I could think of because it will still allow the correct error styling within radios but also if the select is used on its own.